### ISSUE TITLE
Change deployment target depending on branch

### ISSUE THREAD

**Issue by zanona** [Shippable/support#760]
Hey guys, first of all thank you so much for your service. I've tried to use other CI platforms but found Shippable the easier and more reliable to use.

I saw that inside `shippable.yml` file, I can include or exclude branches for being tracked. However, I was thinking if is it currently possible to define different deployment targets on `after_success` depending on the branch?

For example, when pushed to `master` I would like to deploy to the production server, and when pushed to any other branch, let's say, dev… it would be deployed to a different server?
So basically just change the `after_success` argument depending on the branch being validated.

If that is not currently possible, do you recommend any kind of structure to achieve this behaviour?

Thank you in advance
